src/WordContainer.java

```diff
@@ -0,0 +1,7 @@
+import java.util.ArrayList;
```

Comment:
It seems there's no need to separate the WordContainer and WordOperator classes. You'd better merge them.

Reply:
We think word Container is an object and needs a separate class